### PR TITLE
Feature/spi column nrc ranking

### PR DESCRIPTION
**Pull request by weberjavi**
### Description
This PR updates the layout of the SPI ranking.
It also uses info provided by the metadata service on the global biodiversity sidebar cards (priority, rarity and richness).
### Designs
image
### Testing instructions
-  Check the text inside the white info cards on the biodiversity sidebar (on global).
-  get into a National Report Card and check the ranking view.
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-305
https://half-earth-map.atlassian.net/browse/HE-313

### CODE REVIEW

**Comment by gretacv** [comment]
Hi Javi,
The metadata modals have all their title, nice job! :D 
I have seen that there is a _pixel misalignment_ in the ranking, see picture.
Screenshot 2021-04-26 at 12 14 29
And there is a couple of tooltips bugs, maybe for another PR.
1. opacity tooltip in NRC see [HE-302](https://half-earth-map.atlassian.net/browse/HE-302)
2. side bar tooltip, check screenshot. 
Screenshot 2021-04-26 at 12 11 49

**Review by gretacv — APPROVED** [review]
🎉 🚀

File: src/components/local-scene-sidebar/overview-sidebar/local-priority-card/local-priority-card.js [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+      setProtectionsMetadata(data);
+    })
+  }, []);
+
+  useEffect(() => {
+    const md = metadataConfig[COUNTRY_PRIORITY]
+    metadataService.getMetadata(md.slug).then( data => {
+      setPriorityMetadata(data);
+    })
+  }, []);
```
Comment (Bluesmile82) [inline src/components/local-scene-sidebar/overview-sidebar/local-priority-card/local-priority-card.js:29]: Just wondering, why not group them on the same useEffect as they are going to happen on mount?

**Review by Bluesmile82 — COMMENTED** [review]
Looks great 💯

File: src/components/local-scene-sidebar/overview-sidebar/local-priority-card/local-priority-card.js (hunk 1 above)
Comment (weberjavi) [reply src/components/local-scene-sidebar/overview-sidebar/local-priority-card/local-priority-card.js:29]: Just a matter of taste. Having each `effect` on its own function looks cleaner to me 😅

**Pull request merged**